protos/core.proto

```diff
@@ -80,3 +81,11 @@ message BlockHeaders {
 message ChainLockSignatureMessages {
   repeated bytes messages = 1;
 }
+
+message GetEstimatedTransactionFeeRequest {
+
```

Comment:
According to `estimateFee` RPC command, it should accept number of blocks. Correct me if I'm wrong.